R/utils.R

```diff
@@ -206,6 +206,7 @@ calculate_columns_from_existing_data <- function(data) {
       data <- data %>% 
         dplyr::group_by_at(dplyr::vars(dplyr::starts_with("region_level"))) %>%
         dplyr::arrange(date, .by_group = TRUE) %>%
+        tidyr::fill(!!cumulative_count_name) %>% # Fill LOCF for cumulative data
```

Comment:
Nice this covers most of the edge cases. I think their still might be an issue when n = 1 as there is nothing to use for the lag variable (so would need to `fill = 0`)

Reply:
sure - i've set `lag(..., default = 0)` which should address this?